torch/testing/_internal/distributed/rpc/jit/dist_autograd_test.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+import torch
+import torch.distributed.autograd as dist_autograd
+from torch.testing import FileCheck
+from torch.testing._internal.dist_utils import dist_init
+from torch.testing._internal.distributed.rpc.rpc_agent_test_fixture import (
+    RpcAgentTestFixture,
+)
+
+
+@unittest.skipIf(
+    not torch._six.PY3, "Pytorch distributed autograd package does not support python2"
+)
+class JitDistAutogradTest(RpcAgentTestFixture):
```

Comment:
there are test_trainer_ps_torchscript_functions() and test_backward_simple_script_call() as well , those are hard to move though

Reply:
I will leave them there for now. This PR is asked from https://github.com/pytorch/pytorch/pull/33329#discussion_r380962331.